### ISSUE TITLE
Phasmaphobia Rating needs reworking

### ISSUE THREAD

**Issue by Thebro57** [Whisky-App/whisky-book#148]
_Phasmaphobia_, currently rated Platinum, lacks some online multiplayer features when using Whisky. The proximity chat doesn't work at all in lobby, and in-game you can only hear the other people in the lobby and not talk to them (which I believe is a product of the microphone not getting permissions in macOS settings, but you can't change it because _Phasmaphobia_/Whisky/Steam never asks for permissions).

With this in mind, I suggest _Phasmaphobia_ recieving a rating of Silver, because while single player still works, the microphone is still limited and still limits some major aspects of the game.

**Or instead of re-rating it, you can fix it :)**
much preferred

**Comment by mattmaniak** [comment]
Currently we will downgrade Phasmophobia to Silver as mentioned and close this issue when it's done. For a request to fix the microphone please create an issue in the main [Whisky repo](https://github.com/Whisky-App/Whisky/issues). Please look for other issues as there may be such request already like [this](https://github.com/Whisky-App/Whisky/issues/1266). Just to avoid duplicates.

**Comment by Thebro57** [comment]
> Currently we will downgrade Phasmophobia to Silver as mentioned and close this issue when it's done. For a request to fix the microphone please create an issue in the main [Whisky repo](https://github.com/Whisky-App/Whisky/issues). However I see that you have created a similar [issue](https://github.com/Whisky-App/Whisky/issues/1266).

Yes, I believe it's a broader issue because Steam itself isn't detecting microphone input either.  You can read more in the issue in the Whisky repo.
